seguro/common/store.py

```diff
@@ -33,6 +38,9 @@ def __init__(
         secret_key=S3_SECRET_KEY,
         secure=S3_SECURE,
         region=S3_REGION,
+        log_level=LOG_LEVEL,
+        log_max_bytes=MAX_BYTES,
+        log_backup_count=BACKUP_COUNT,
```

Comment:
If we handle this via S3, the log retention would be handled via S3 object lifecycle rules:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html

Reply:
We keep those parameters to still be able to instantiate a `file_logger`